homeassistant/components/zha/sensor.py

```diff
@@ -208,6 +208,12 @@ def formatter(value):
             state_attrs["battery_quantity"] = battery_quantity
         return state_attrs
 
+    def async_update_state_attribute(self, key, value):
+        """Update a single device state attribute."""
+        if key == "battery_voltage":
+            self._device_state_attributes["voltage"] = f"{round(value/10, 1)}V"
```

Comment:
We shouldn't include a unit of measurement in state attributes. Units can be documented if needed.

We prefer to report measurements as individual sensors where they each can have a unit of measurement defined.

Reply:
Yep, I just saw similar comment for another PR. I'll remove the UoM from this one. ATM can't implement multiple entities per single ZCL cluster, not until we refactor zha device discovery which is in the works.

Reply:
That's good! Thanks!